cabal.project

```diff
@@ -52,4 +52,42 @@ if impl(ghc >= 9.10)
 
   constraints:
     -- Earlier versions do not compile with ghc-9.10
-    , plutus-ledger-api ^>=1.31
+    , plutus-ledger-api ^>=1.32
```

Comment:
This is not necessary because you are specifying a SRP which will already lock the plutus version